packages/components/src/components/SquiggleViewer/ValueWithContextViewer.tsx

```diff
@@ -69,23 +70,31 @@ export const ValueWithContextViewer: FC<Props> = ({ value }) => {
 
   const isRoot = path.isRoot();
 
-  // This doesn't just memoizes the defaults, but also affects children, in some cases.
-  const defaults: LocalItemState = useMemo(() => {
-    // TODO - value.size() would be faster.
-    const childrenElements = getChildrenValues(value);
+  // Collapse children and element if desired. Uses crude heuristics.
+  useState(() => {
+    const tagsDefaultCollapsed = new Set(["Bool", "Number", "Void", "Input"]);
+    // TODO - value.size() could be faster.
```

Comment:
Btw, I don't understand this comment (copied it from before). value doesn't directly have a size() function.